core/state-db/src/lib.rs

```diff
@@ -32,16 +32,16 @@
 //!
 // end::description[]
 
-#[macro_use] extern crate log;
-#[macro_use] extern crate parity_codec_derive;
+#[macro_use]extern crate log;
```

Comment:
Nitpick: Either move the `macro_use` above or re-add the whitespace.

Reply:
done